packs/rollable-tables/1st-level-consumables.json

```diff
@@ -1,8 +1,8 @@
 {
     "_id": "tlX5PLwar8b1tmiQ",
-    "description": "Table of 1st-Level Consumables",
+    "description": "",
```

Comment:
```suggestion
    "description": "Table of 1st-Level Consumables",
```

Reply:
Thanks for your patience and help. I'll get it right next time.